Postgres/18.6/scripts/migrate-docker-entrypoint.sh

```
@@ -0,0 +1,143 @@
+        fi
+         gosu postgres pg_checksums --check -D "$PGDATAOLD" &> /dev/null || gosu postgres pg_checksums --disable --pgdata "$PGDATANEW"
+	fi
+
+    rm -rf "$PGDATANEW/pg_upgrade_output.d"
+    if ! gosu postgres pg_upgrade --link; then
+        echo "Failed to upgrade the server"
+        find "$PGDATANEW/pg_upgrade_output.d" -type f -name 'pg_upgrade_server.log' -exec cp -t "$PGMOUNT" {} +
+        cat "$PGMOUNT/pg_upgrade_server.log"
+        if [ -f "$PGDATAOLD/PG_VERSION" ] && [ -f "$PGDATANEW/PG_VERSION" ]; then
+            echo "Cleaning up the new cluster"
+            rm -r $PGDATANEW/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)